addons/website_slides/controllers/main.py

```diff
@@ -444,6 +444,8 @@ def channel(self, channel, category=None, tag=None, page=1, slide_type=None, unc
             # chatter
             'rating_avg': channel.rating_avg,
             'rating_count': channel.rating_count,
+            'review_count': channel.review_count,
+            'reviews_domain': [('body', '!=', '<p></p>'), ('body', '!=', '')],
```

Comment:
Mmmh, this is a side effect of having a void body containing in messages I suppose... We had the issue in some other places. Why not defining an body_is_empty computed / searchable field on mail.message (not stored) ? That way domain would be [('body_is_empty', '=', False)], and search method will simply transform it into your domain.

Purpose of that change is to hide the details of "is empty" in a tool search field, while finding a proper solution to this issue. What do you think ?

Reply:
I think it's nice solution for the issue here. However, doesn't the compute field need to be stored in order to use it for a search ?

Reply:
Not necessarily, you can define the _search attribute. See in existing code, but in short, you have to code a method that returns a domain for a search. It is therefore only a shortcut, but allows to hide the complexity on message model itself.